compiler-core/src/language_server/tests.rs

```diff
@@ -601,15 +607,15 @@ impl<'a> TestProject<'a> {
     pub fn at<T>(
         &self,
         position: Position,
+        new_src: Option<&str>,
```

Comment:
Remove this please. The other tests should not change, we don't want to have a parameter that every test needs to provide but only this one uses.